knowledge-base/grid-custom-filter-menu.md

```diff
@@ -0,0 +1,209 @@
+    new Country { Id = 4, CountryName = "Austria" },
+    new Country { Id = 5, CountryName = "Azerbaijan" },
+    new Country { Id = 6, CountryName = "Belarus" },
+    new Country { Id = 7, CountryName = "Belgium" },
+    new Country { Id = 8, CountryName = "Bosnia & Herzegovina" },
+    new Country { Id = 9, CountryName = "Bulgaria" },
+    new Country { Id = 10, CountryName = "Croatia" },
+    new Country { Id = 11, CountryName = "Cyprus" },
+    new Country { Id = 12, CountryName = "Czech Republic" },
+    new Country { Id = 13, CountryName = "Denmark" },
+    new Country { Id = 14, CountryName = "Estonia" },
+    new Country { Id = 15, CountryName = "Finland" },
+    new Country { Id = 16, CountryName = "France" },
+    new Country { Id = 17, CountryName = "Georgia" },
+    new Country { Id = 18, CountryName = "Germany" },
+    new Country { Id = 19, CountryName = "Greece" },
+    new Country { Id = 20, CountryName = "Hungary" },
+    new Country { Id = 21, CountryName = "Iceland" },
+    new Country { Id = 22, CountryName = "Ireland" },
+    new Country { Id = 23, CountryName = "Italy" },
+    new Country { Id = 24, CountryName = "Kosovo" },
+    new Country { Id = 25, CountryName = "Latvia" },
+    new Country { Id = 26, CountryName = "Liechtenstein" },
+    new Country { Id = 27, CountryName = "Lithuania" },
+    new Country { Id = 28, CountryName = "Luxembourg" },
+    new Country { Id = 29, CountryName = "Macedonia" },
+    new Country { Id = 30, CountryName = "Malta" },
+    new Country { Id = 31, CountryName = "Moldova" },
+    new Country { Id = 32, CountryName = "Monaco" },
+    new Country { Id = 33, CountryName = "Montenegro" },
+    new Country { Id = 34, CountryName = "Netherlands" },
+    new Country { Id = 35, CountryName = "Norway" },
+    new Country { Id = 36, CountryName = "Poland" },
+    new Country { Id = 37, CountryName = "Portugal" },
+    new Country { Id = 38, CountryName = "Romania" },
+    new Country { Id = 39, CountryName = "Russia" },
+    new Country { Id = 40, CountryName = "San Marino" },
+    new Country { Id = 41, CountryName = "Serbia" },
+    new Country { Id = 42, CountryName = "Slovakia" },
+    new Country { Id = 43, CountryName = "Slovenia" },
+    new Country { Id = 44, CountryName = "Spain" },
+    new Country { Id = 45, CountryName = "Sweden" },
+    new Country { Id = 46, CountryName = "Switzerland" },
+    new Country { Id = 47, CountryName = "Turkey" },
+    new Country { Id = 48, CountryName = "Ukraine" },
+    new Country { Id = 49, CountryName = "United Kingdom" },
+    new Country { Id = 50, CountryName = "Vatican City" }
+};
+    public class Country
+    {
+        public int Id { get; set; }
+        public string CountryName { get; set; } = null!;
+    }
+}
+````
+````CustomFilterMenu.razor
+@using Telerik.DataSource
+@using Microsoft.AspNetCore.Components
+@using static Home
+
+<label for="filterOperator"><b>Select Filter Operator for the AutoComplete:</b></label>
```

Comment:
Use `<strong>` instead of `<b>`. The former is semantic markup, which implies importance, while the latter is just a style.